src/index.d.ts

```diff
@@ -10,7 +10,7 @@ import {Locale} from './locale';
 import {Theme, ThemePrimitives} from './theme';
 
 type UseStyletronFn<Theme> = () => [(arg: StyleObject) => string, Theme];
-export function createThemedUseStyletron<Theme>(): UseStyletronFn<Theme>;
+export function createUseStyletronThemed<Theme>(): UseStyletronFn<Theme>;
```

Comment:
Please maintain the existing function declarations and create new aliases for the names

Reply:
Can you please let me know what exactly do you mean by aliasing?